bsc/0x10ED43C718714eb63d5aA57B78B54704E256024E.json

```diff
@@ -0,0 +1,961 @@
+{
+  "name": "PancakeSwap Router v2",
+  "chainId": "56",
+  "address": "0x10ED43C718714eb63d5aA57B78B54704E256024E",
+  "metadata": {
+    "abi": [
```

Comment:
the abi field should be included in the output field like:

```
{
  "metadata":{
      "output": {
           "abi": []
      }  
   }
}

```

Reply:
Oh right, good catch 😅 
Fixed it